derives/Cargo.toml

```diff
@@ -4,9 +4,13 @@ version = "0.0.10"
 authors = ["Lloyd <lloydmeta@gmail.com>"]
 description = "frunk_derives contains the custom derivations for certain traits in Frunk."
 license = "MIT"
-homepage = "https://github.com/lloydmeta/frunk"
+documentation = "https://docs.rs/frunk_derives/0.0.10/frunk_derives/"
+repository = "https://github.com/lloydmeta/frunk"
 keywords = [ "HList", "Generic", "derives"]
```

Comment:
I know you didn't touch this, but maybe we can either delete Hlist and Generic from here or get rid of keywords altogether.